src/Components/Article_comments.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { fetchComments } from "../Utils/Api-requests";
+
+const ArticleComments = () => {
+  const [backendComments, setBackendComments] = useState([]);
+  const [showComments, setShowComments] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [err, setErr] = useState(null);
+  const { article_id } = useParams();
+  useEffect(() => {
+    fetchComments(article_id)
+      .then((commentData) => {
+        setBackendComments(commentData.comments);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setErr(
+          "Sorry something went wrong, could not load comments. please try again"
+        );
+      });
+  }, [article_id]);
+
+  const handleShowComments = () => {
+    setShowComments((showComments) => {
+      if (showComments) {
+        return false;
+      } else {
+        return true;
+      }
+    });
+  };
+
+  if (err) return <p className="articlecomments__error-message">{err}</p>;
+  if (isLoading) {
+    return <p>Loading...</p>;
+  } else {
+    return (
+      <>
+        <div className="commentsbutton__parent">
+          <button
+            className="commentsbutton__child"
+            onClick={handleShowComments}
+          >
+            Comments
+          </button>
+          {showComments && (
+            <div className="articlecomments">
+              {backendComments.map((articleComment) => {
+                return (
+                  <li key={articleComment.article_id}>
+                    <p className="articlecomments__comment_author">
+                      {" "}
+                      Author :{articleComment.author}
+                    </p>
+                    <p className="articlecomments__comment_body">
+                      {articleComment.body}
+                    </p>
+                    <p className="articlecomments__comment_votes">
+                      {" "}
+                      Votes: {articleComment.votes}
+                    </p>
+                    <p className="articlecomments__comment_created_at">
+                      {" "}
+                      Created at: {articleComment.created_at}
+                    </p>

```

Comment:
The formatting of this date is a bit confusing and hard to read for a user (currently appears like 2020-11-22T11:13:00.000Z). Can you format it to make it look a bit more user friendly?